src/main/java/uk/gov/justice/digital/hmpps/csr/api/domain/ActionType.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.csr.api.domain
+
+enum class ActionType(val action: String, val number: Int) {
+    UNCHANGED("UNCHANGED", 0),
+    ADD("ADD", 1),
+    EDIT("EDIT", 2),
+    DELETE("DELETE", 3);
+
```

Comment:
The values need to match with the enums in here:
https://github.com/ministryofjustice/cmd-api/blob/main/src/main/java/uk/gov/justice/digital/hmpps/cmd/api/domain/ShiftActionType.kt

It might be useful to have them named the same, we should change one or the other or both.

Reply:
These values are matching the c# app: 
https://github.com/ministryofjustice/prison-officer-diary-api/blob/master/src/PrisonOfficer.Diary.Core/Models/Business/NotificationActionType.cs

Correct me if I am wrong on any of this:
I'm working on the assumption that the ints are stored in the database, and the string version are our own interpretation of those. I'll need to updatethe action/String to match the `cmd-api` casing but I dont think that we need the descriptions here.

Reply:
did you update this since I last commented? the values look fine.
the 0,1,2,3 matches whats in the DB and the add edit etc matches what the client (CMD-API) is expecting.
you're right we don't need the descriptions.

Reply:
yaha, I did my suggested change, the strings were all caps previously :)